freqtrade/data/history/hdf5datahandler.py

```diff
@@ -157,7 +189,11 @@ def trades_append(self, pair: str, data: TradeList):
         """
         raise NotImplementedError()
 
-    def _trades_load(self, pair: str, timerange: Optional[TimeRange] = None) -> TradeList:
+    def _trades_load(
+        self,
+        pair: str,
+        timerange: Optional[TimeRange] = None,
+    ) -> TradeList:
```

Comment:
Please don't make pointless formatting changes.

I think i mention this in every other PR - please keep this in mind - i'm tired of repeating my self over and over and over again with the same thing, and this makes reviewing VERY tiresome.

Reply:
I had previously added an argument which brought this line past 100 characters, later I decided I didn't need it and removed it (with VSCodes replace tool probably), but didn't remove the formatting change. 

Please be understanding that I may not catch everything every time I push

Reply:
That's for sure true ... but in my understanding from the above comment, you did want an opinion ... which requires an "early" review.